coalib/bears/requirements/PythonRequirement.py

```diff
@@ -5,6 +5,37 @@ class PythonRequirement(PackageRequirement):
     """
     This class is a subclass of ``PackageRequirement``, and helps specifying
     requirements from ``pip``, without using the manager name.
+
```

Comment:
Under `def multiple(...)`. You can do:

```
 +        Should not be instances of ``PackageRequirement``, as this is an
 +        abstract class:
 +
 +        >>> PackageRequirement.multiple(('pip', 'coala_decorators', '0.1.0'),)
 +        Traceback (most recent call last):
 +        ...
 +        NotImplementedError
 +
 +        It can only be used for requirements of the same manager. For example,
 +        consider a manager ``XYZRequirement`` that inherits from
 +        PackageRequirement:
 +        >>> class XYZRequirement(PackageRequirement): pass
+    This is the case where you would provide strings only, to specify the
 +    latest version automatically:
 +
 +    >>> REQUIREMENTS = XYZRequirement.multiple(
 +    ...     'package1', 'package2')
 +
 +    And if you choose to mix them, specifying version for some and for some
 +    not:
 +
 +    >>> REQUIREMENTS = XYZRequirement.multiple(
 +    ...     'package1', ('package2', '2.0'))
 +
 +    Lists are also valid arguments:
 +
 +    >>> REQUIREMENTS = XYZRequirement.multiple(
 +    ...     ['package1', '1.0'],)
 +
 +    In case you provide too many arguments into the tuple, an error will be
 +    raised:
 +
 +    >>> REQUIREMENTS = XYZRequirement.multiple(
 +    ...     'package1', ('package2', '2.0', '# some comment'))
 +    Traceback (most recent call last):
 +    ...
 +    TypeError: Too many elements provided.
```